Makefile

```diff
@@ -42,3 +42,52 @@ build:
 .PHONY: clean
 clean:
 	rm -rf integration/testdata/fixtures/
+
+$(GOBIN)/bbolt:
+	go get -v github.com/etcd-io/bbolt/cmd/bbolt
+
+DB_TYPE ?= trivy
+
+ifeq ($(DB_TYPE),trivy-light)
+	DB_ARG := --light
+else
+	undefine DB_ARG
+endif
+
+trivy-db:
+	make build
```

Comment:
Is this necessary?

Reply:
This creates a non phony target, so I can ensure that trivy-db as a binary has been built jn later steps.